tests/build/build.go

```diff
@@ -1716,3 +1940,25 @@ func cleanupWebhooks(f *framework.Framework, repoName string) {
 		}
 	}
 }
+
+// createGitlabBuildSecret creates a secret of gitlab build
+func createGitlabBuildSecret(f *framework.Framework, secretName string, annotations map[string]string, token string) error {
```

Comment:
I see another similar function [here](https://github.com/konflux-ci/e2e-tests/blob/6edc95001438142e3661c8c699274052573d0f72/tests/integration-service/gitlab-integration-reporting.go#L274) ... will it be good to move this function to [utils/build](https://github.com/konflux-ci/e2e-tests/tree/main/pkg/utils/build) pkg ? so that we can reuse in multiple places